### ISSUE TITLE
AC922 CS22B RH86 MOFED firmware install requires force option.

### ISSUE THREAD

**Issue by markprez** [xcat2/xcat-core#7228]
Our CS21B service pack with RH84 shipped firmware 16.32.1010 with MOFED MOFED 4.9-4.1.1.1
For RH86, NVIDIA delivered MOFED 4.9-5.1.0.2 with f.w 16.31.1354.

In a diskless environment we would run updatenode <node> -P "update_mlnx_adapter_firmware /install/REPO/software/mellanox/ofed/iso/redhat/8.6/ppc64le/MLNX_OFED_LINUX-4.9-5.1.0.2-rhel8.6-ppc64le.iso

In a diskfull environment when we install MOFED it would update the firmware.

Unfortunately, since 16.31.1354 is an earlier f.w version than 16.32.1010, our scripts will not install it. NVIDIA requested we use 16.31.1354 with 4.9-5.1.0.2.

To get around this I created a script update_mlnx_adapter_firmware.force  to run against all nodes.

```
diff update_mlnx_adapter_firmware.force update_mlnx_adapter_firmware
44c44
< ${MOUNT_DIR}/mlnxofedinstall --fw-update-only --force-fw-update
---
> ${MOUNT_DIR}/mlnxofedinstall --fw-update-only
```

I run this on all diskless and diskfull nodes as part of my upgrade.
updatenode <noderange> -P -P "update_mlnx_adapter_firmware.force /install/REPO/software/mellanox/ofed/iso/redhat/8.6/ppc64le/MLNX_OFED_LINUX-4.9-5.1.0.2-rhel8.6-ppc64le.iso"

This could be handled with a -f flag on the original update_mlnx_adapter_firmware.

**Comment by besawn** [comment]
@gurevichmark Can you please add an option to force the firmware update to `update_mlnx_adapter_firmware`?

**Comment by gurevichmark** [comment]
Fixed by #7229
